src/OrleansRuntime/Streams/PubSub/PubSubRendezvousGrain.cs

```diff
@@ -166,7 +166,7 @@ private static bool IsDeadProducer(IStreamProducerExtension producer)
             PubSubSubscriptionState pubSubState = State.Consumers.FirstOrDefault(s => s.Equals(subscriptionId));
             if (pubSubState == null)
             {
-                pubSubState = new PubSubSubscriptionState(subscriptionId, streamId, streamConsumer, filter);
```

Comment:
Would you mind removing the filter from the PubSubSubscriptionState constructor.  There are no scenarios where setting this during construction is desired.